plugin/trino-thrift/src/test/java/io/trino/plugin/thrift/integration/TestThriftConnectorTest.java

```diff
@@ -44,4 +44,21 @@ public void testShowSchemas()
                 .row("sf1");
         assertContains(actualSchemas, resultBuilder.build());
     }
+
+    @Test
+    public void testCreateArrayTable()
```

Comment:
Add these in a new commit since they are new tests not part of either the existing smoke test or the distributed queries test.